src/transformers/feature_extraction_common_utils.py

```diff
@@ -395,21 +367,22 @@ def get_feature_extractor_dict(
         return feature_extractor_dict, kwargs
 
     @classmethod
-    def from_dict(cls, feature_extractor_dict: Dict[str, Any], **kwargs) -> "PreTrainedFeatureExtractor":
+    def from_dict(cls, feature_extractor_dict: Dict[str, Any], **kwargs) -> PreTrainedFeatureExtractor:
         """
-        Instantiates a :class:`~transformers.PreTrainedFeatureExtractor` from a Python dictionary of parameters.
+        Instantiates a :class:`~transformers.PreTrainedSequenceFeatureExtractor` from a Python dictionary of
+        parameters.
```

Comment:
This change contradicts the type. I think it should still say it will return a `PreTrainedFeatureExtractor`, as that's a mixin that would be used for sequences and images, right?

Reply:
Hmm, I was thinking more that we will take the approach of the `generation_utils.py` docstring-like: https://github.com/huggingface/transformers/blob/917f1045025001bc7dfefc992bd101e02fe4bada/src/transformers/generation_utils.py#L769 so that we'll add `... or` statements. Since it's a mixin and doesn't have a `__init__(...)` method I thought, it has to create one of either `PreTrainedSequenceFeatureExtractor` or `PreTrainedImageFeatureExtractor`. Also, regarding the docstring I think it's nicer if the user can directly click on either `PreTrainedSequenceFeatureExtractor` or `PreTrainedImageFeatureExtractor` since the important functionalities are to be found there...

But not sure here...what do you think @LysandreJik @sgugger ?